src/main/resources/META-INF/plugin.xml

```diff
@@ -85,6 +85,11 @@
                     icon="MacheteIcons.SLIDE_OUT">
                 <override-text place="GitMacheteContextMenu"/>
             </action>
+
+            <action class="com.virtuslab.gitmachete.frontend.actions.contextmenu.ShowSelectedBranchInGitLogAction"
```

Comment:
Also! pls bump version to 1.3.0 (since this PR adds a new user-facing feature) + update CHANGE-NOTES.html; also pls mention the recently merged fix wrt. pull/fetch logic

Reply:
(version.gradle still to be updated)

Reply:
version is 1.3.0, isnt that right?

Reply:
Aww crap, check a wrong commit, sorry